src/main/java/net/minecraftforge/server/permission/IPermissionHandler.java

```diff
@@ -44,7 +44,7 @@
 
     /**
      * @param node Permission node
-     * @return Description of the node
+     * @return Description of the node. "" in case this node doesn't have a decription
```

Comment:
Can't we just put @nonnull on the method and leave it at that? The universal nonnull empty value for the string type is the empty string and programmers should be able to figure that much out :P